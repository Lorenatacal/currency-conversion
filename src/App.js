import React from 'react';
import CurrencyConversion from './CurrencyConversion';
import styled from 'styled-components';

const Title = styled.h2`
  text-align: center;
`;
function App () {
  return (
    <div className="App">
      <Title>Currency Conversion</Title>
      <CurrencyConversion />
    </div>
  );
}

export default App;
