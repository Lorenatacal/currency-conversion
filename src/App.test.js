import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

Enzyme.configure ({adapter: new Adapter ()});

it ('renders without crashing', () => {
  const div = document.createElement ('div');
  ReactDOM.render (<App />, div);
  ReactDOM.unmountComponentAtNode (div);
});

test ('App should render correctly', () => {
  const wrapper = Enzyme.shallow (<App />);
  expect (toJson (wrapper)).toMatchSnapshot ();
});
