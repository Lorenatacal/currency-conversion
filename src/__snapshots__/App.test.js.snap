// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="App"
>
  <styled.h2>
    Currency Conversion
  </styled.h2>
  <CurrencyConvertor />
</div>
`;
